src/docs/truffle/reference/configuration.md

```diff
@@ -51,6 +51,137 @@ Any of the following solutions will remedy this issue:
 
 ## General options
 
+### SolidityLog
+
+Like JavaScript's `console.log`, you can utilize the same concept to print
+state of your contract during execution. This log will appear when you use
```

Comment:
```suggestion
the state of your contract during execution. This log will appear when you use
```

Reply:
I want to say, print state, not the state.